ui/src/data_explorer/components/SendToDashboardOverlay.tsx

```diff
@@ -211,8 +233,16 @@ class SendToDashboardOverlay extends PureComponent<Props, State> {
 
     const legend = isStaticLegend ? STATIC_LEGEND : {}
 
+    let selectedDashboards = [...this.selectedDashboards]
```

Comment:
do you need to spread this.selectedDashboards here? could this not be just `selectedDashboards = this.selectedDashboards`?